### PR TITLE
EmailInvitationRequest

### PR DESCRIPTION
**Pull request by jay-lee00**
re #5252

### CODE REVIEW

File: SnapMD.ConnectedCare.ApiModels/EmailInvitationRequest.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SnapMD.ConnectedCare.ApiModels
+{
+    public class EmailInvitationRequest
+    {
+        public int? HospitalId { get; set; }
+
+        public int? UserId { get; set; }
```
Comment (lorddev) [inline SnapMD.ConnectedCare.ApiModels/EmailInvitationRequest.cs]: @jay-lee00 Why do we need hospital ID and user ID?

Comment (jay-lee00) [inline SnapMD.ConnectedCare.ApiModels/EmailInvitationRequest.cs]: There is an older api sending invitation to the couser.  I use this for
both api request.
https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail
This
email has been sent from a virus-free computer protected by Avast.
www.avast.com
https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail
<#DDB4FAA8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by lorddev** [comment]
@jay-lee00 Please create a separate class.

**Comment by lorddev** [comment]
@jay-lee00 Can you look at the merge conflicts?

**Pull request merged**